webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: path.join(__dirname, 'src', 'index.js'),
+  output: {
+    path: path.resolve(__dirname, 'dist'),
```

Comment:
If we add the `filename` property, would this allow us to pre-populate `src/index.html` with the corresponding script tag? `<script type="text/javascript" src="main.js"></script>`

Just suggestion, please leave as-is if you think that's a better approach!
```suggestion
    filename: 'main.js',
```

Reply:
I think that this is a good idea. Will add to talking points when backend team reconvenes. Thank you Olivia!

Reply:
Olivia/Khai: this is not needed. The `HtmlWebpackPlugin` I included on line 32 already adds the bundle script to `index.html`.

Here is an example of the resulting `index.html` from the webpack build:
![Screen Shot 2021-12-09 at 4 49 25 PM](https://user-images.githubusercontent.com/16950863/145481269-82fbaa0f-c55d-47fa-9d01-6af42619b0ff.png)